omnibus/Gemfile

```diff
@@ -2,11 +2,13 @@
 source 'https://rubygems.org'
 
 # Install omnibus
-gem 'omnibus', '~> 5.3'
+# Sadly due to an ongoing msys2 issue, producing windows builds requires
+# the two branches below.
+gem 'omnibus', github: 'chef/omnibus', branch: 'ksubrama/gcc_investigate'
```

Comment:
Long story...  has to do with the back-and-forth we've had on https://github.com/chef/omnibus/pull/695 where some of the debug features that I wanted seem like excess baggage (i.e. code to solve problems that shouldn't exist in the first place...  but problems that do exist now...  problems that I don't have time to solve properly).  But mixed in that PR is a bunch of critical fixes too...  so client pool is taking care of extracting the important stuff out and making sure the tests pass and all that good jazz.  Until then, even chef-client and chef-dk are pinned to these branches :(